README.md

```diff
@@ -1,3 +1,25 @@
-# Parallelisation of Graph Algorithms in Functional Programming Languages
+# Paralelizacija grafovskih algoritmov v funkcijskih programskih jezikih
 
-In the scope of my bachelor's thesis I'll research parallelisation of graph algorithms in functional programming languages. Text is written in Slovene.
+V okviru diplomske naloge bom raziskoval paralelizacijo grafovskih algoritmov v funkcijskih programskih jezikih.
+
+## Nastavitev okolja
+
+Za poganjanje projekta se zahteva package manager **opam** (inštalacija: https://opam.ocaml.org/doc/Install.html) ter knjižnice, zapisane v **requirements.txt**. Po naloženem opamu, poženite `opam install .` za nalaganje potrebnih modulov.
+
+
+## Struktura projekta
+
+V folderju /diploma so shranjeni pisni izdelki diplomske naloge (najpomembnejši izmed njih je verjetno [diplomska naloga sama](https://github.com/tjazerzen/parallelisation-of-graph-algorithms-in-functional-programming-languages/blob/master/diploma/dokument-diplomska-naloga.pdf)). Mapa /diploma uporablja rezultate iz mape /playground.
```

Comment:
`/diploma` in podobno za ostale poti.

Reply:
b9c32cd0d7c41ebb0fba09e79cfbd1670f881bb3.